### ISSUE TITLE
Why is the intro to CC before the comand line?

### ISSUE THREAD

**Issue by orchid00** [datacarpentry/cloud-genomics#15]
Is there any particular reason to do the introduction to Cloud computing for genomics (module 3) before introduction to command line (module 4)? Personally, and discussing this with others the command line lesson should be before.
By reading in the contents for Cloud computing (CC) module 3, I found that in
 cloud-genomics/lessons/4.parallel-analysis.md
it says: As we learned in the Unix shell lesson, 
However that is not until the next module. 
Should this order be re-scheduled?

**Comment by devbioinfoguy** [comment]
Yes, looking back at these lessons now, #3 and #4 don't really make sense. I think these were developed as stop-gap measures many months ago. These need careful examination, and perhaps decisions from leadership on how to move these forward.

That being said, in a workshop, instructors can always do 0 & 1 from here, then run Intro to Shell, then come back for #2 or #5 later on.

More later...
